library/patientvalidation.inc.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Need a standard header. See here for an example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock